core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -300,7 +300,7 @@ public static void annotateAlteration(Alteration alteration, String proteinChang
 
         proteinChange = proteinChange.trim();
 
-        Pattern p = Pattern.compile("([A-Z]?)([0-9]+)(_[A-Z]?([0-9]+))?(delins|ins|del)([A-Z0-9]*)", Pattern.CASE_INSENSITIVE);
+        Pattern p = Pattern.compile("([A-Z]?)([0-9]+)(_[A-Z]?([0-9]+))?(delins|ins|del)([A-Z0-9\\*]*)", Pattern.CASE_INSENSITIVE);
```

Comment:
Adding the stop codon (*) in group 6 will match patterns that have multiple asterisks, which is not valid.